### DISCUSSION TITLE
Players slots serialized

### DISCUSSION THREAD

**Discussion by XDSenDX** [Vdauphin/HeartsAndMinds#1574] [Question and answer] [answered]
Hello i took a look at save.sqf, i wonder how does player slots from L227-242 work?

Cheers.

**Comment by Vdauphin** [comment, marked as answer]
Hello,
Look are https://forums.bohemia.net/forums/topic/165948-mp-btc-hearts-and-minds/?page=75&tab=comments#comment-3469051
Cheers

**Reply by XDSenDX** [reply]
Thanks for this.

**Reply by XDSenDX** [reply]
For the mission settings is it located in param.hpp?

**Reply by Vdauphin** [reply]
Yes

**Reply by XDSenDX** [reply]
> Yes

Which setting to be exact? I've spent quite a while but couldn't find them?

Sorry for this.

**Reply by Vdauphin** [reply]
https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L134-L139

**Reply by XDSenDX** [reply]
> https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/core/def/param.hpp#L134-L139

Bravo!
